tests/layer_tests/mo_python_api_tests/test_mo_convert_paddle.py

```diff
@@ -77,7 +77,7 @@ def create_paddle_static_module(tmp_dir):
     pd_model, x, y = make_pd_static_graph_model(shape)
     ref_model = make_ref_graph_model(shape)
 
-    return pd_model, ref_model, {"example_input": [x], "example_output": [y]}
+    return pd_model, ref_model, {"example_input": [x], "output": [y]}
```

Comment:
OVC python API tests were moved to tests/layer_tests/**ovc**_python_api_tests/
So this change should be done here: https://github.com/openvinotoolkit/openvino/blob/600c2d82836ef9f9443b7db9e8d6502a71c4fcc3/tests/layer_tests/ovc_python_api_tests/test_paddle.py#L80